components/ai_chat/core/browser/ai_chat_service.cc

```diff
@@ -809,20 +803,25 @@ void AIChatService::OnClientConnectionChanged(ConversationHandler* handler) {
   MaybeUnloadConversation(handler);
 }
 
-void AIChatService::OnConversationTitleChanged(ConversationHandler* handler,
-                                               std::string title) {
-  auto conversation_it = conversations_.find(handler->get_conversation_uuid());
-  CHECK(conversation_it != conversations_.end());
-  auto& conversation = conversation_it->second;
-  conversation->title = title;
+void AIChatService::OnConversationTitleChanged(
+    const std::string& conversation_uuid,
+    const std::string& new_title) {
```

Comment:
should this just take a `std::string` so we can move the `new_title` into this function?

Reply:
as discussed in the slack thread, doesn't seem like it's worth the optimization